docs/security/01-01-security.md

```diff
@@ -20,13 +20,16 @@ See [this](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) docume
 
 >**NOTE:** You cannot define groups for the static user store. Instead, bind the user directly to a role or a cluster role by setting the user as the subject of a RoleBinding or ClusterRoleBinding.
 
-By default, there are five roles used to manage permissions in every Kyma cluster. These roles are:
+By default, there are six roles used to manage permissions in every Kyma cluster. These roles are:
   - **kyma-essentials**
   - **kyma-view**
+  - **kyma-namespace-admin**
   - **kyma-edit**
   - **kyma-developer**
   - **kyma-admin**
 
 For more details about roles, read [this](#details-roles-in-kyma) document.
 
 >**NOTE:** The **Global permissions** view in the **Settings** section of the Kyma Console UI allows you to manage bindings between user groups and roles.
```

Comment:
It only allows to manage global permissions (cluserroles, clusterrolebindings).  For managing namespace-wide permissions there is a **Permissions** view in the **Configuration** section of a namespace.

Reply:
fixed